examples/burial-form/ClaimantContactInformation.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {AddressField, EmailField, Page, TextField} from '@department-of-veterans-affairs/va-forms-system-core';
+import {useFormikContext} from "formik";
+import PhoneField from "@department-of-veterans-affairs/va-forms-system-core/form-builder/PhoneField";
```

Comment:
You can import PhoneField like this: 

`import { PhoneField } from "@department-of-veterans-affairs/va-forms-system-core"`

Reply:
Updated!